### PR TITLE
Added regions per Jira ticket

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/biganimal/release/overview/03a_region_support/index.mdx [hunk 1]
```diff
@@ -152,6 +152,30 @@ When using Google Cloud, you can create clusters in the following regions.
...
+| ------------------------ | -------------- |
+| US East (Virginia)       | eastus2        |
+| Canada (Central)         | canadacentral  |
+
+
+#### Asia and Pacific (APAC) 
+
+|       Cloud region       |   Short name   |
+| ------------------------ | -------------- |
+| Asia Pacific (Mumbai)    | india-west     |
```
[the remark this answers is not in the document]
Comment (valeriodelsarto) [reply product_docs/docs/biganimal/release/overview/03a_region_support/index.mdx:171]: this is the region that has been enabled in the end, see https://edb-starlight.slack.com/archives/C02ECM0EL6N/p1695915501687139?thread_ts=1695911709.504079&cid=C02ECM0EL6N

**Review by valeriodelsarto — COMMENTED** [review]
Added a suggestion/comment, all the rest LGTM!

**Comment by nidhibhammar** [comment]
@valeriodelsarto, I have requested your re-review as it is needed to merge the PR and publish the changes. Let me know if any change is still required.

**Comment by ccestes** [comment]
These changes have been merged into #4858. Closing the PR.

**Pull request closed without merging**